recipes/modestpy/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "0.0.7" %}
+
+package:
+  name: modestpy
+  version: {{ version }}
+
+source:
+  fn: modestpy-{{ version }}.tar.gz
+  url: https://github.com/sdu-cfei/modest-py/archive/v{{ version }}.tar.gz
+  sha256: 687d8372540aeeb901c99dc345a1441a0c5d15874aaa365e1754353f90dca075
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
```

Comment:
It looks like only `pip`, `python` and `setuptools` are needed in the `host` section since none of the other packages are imported directly by the `setup.py` or as a side-effect.

Reply:
so ? is it ok ?

Reply:
It passes the tests as is, but in terms of finalizing the recipe, I think we should remove stuff that isn't necessary.